### PR TITLE
add more emphasis, address comments in issues

### PR DESCRIPTION
**Pull request by bendhouseart**
- Addresses issue #4 
- Adds small fixes noted in issue #5

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -1,9 +1,31 @@
 # BIDS Extension Proposals: A Guide
 
+## What is a BIDS Extension Proposal \(BEP\)?
```
Comment (ericearl) [inline docs/guide.md:3]:
```suggestion
## What is a BIDS Extension Proposal (BEP)?
```

File: docs/guide.md [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+
+A BEP, as the name implies, is a method of expanding the BIDS Specification to 
+encompass new features or imaging types. 
+
+Why are they called BEPs? BEP's are modeled after 
+  [Python Extention Proposals](https://peps.python.org/pep-0001/#what-is-a-pep) (PEPs) as
+  they have been an effective community tool to change (either by updating or supplementing) 
+  the Python programming language and our adopting a similar structure (has so far) been an
+  effective method of contributing new changes to BIDS.
```
Comment (ericearl) [inline docs/guide.md:12]:
```suggestion
Why are they called BEPs? BEPs are modeled after 
[Python Extension Proposals](https://peps.python.org/pep-0001/#what-is-a-pep) (PEPs) as
they have been an effective community tool to change (either by updating or supplementing) 
the Python programming language and BIDS contributors adopting a similar structure (has so far) been an
effective method of contributing new changes to BIDS.
```

File: docs/guide.md [hunk 3]
```diff
@@ -1,9 +1,31 @@
...
+
+BEPs have grown the specification to include beyond MRI as well
+as new techniques, and files types and descriptors. A list of completed 
+BEPS can be seen [here](https://bids.neuroimaging.io/get_involved.html#completed-beps) and 
+a list of proposed/on going BEPs can be found [here]https://bids.neuroimaging.io/get_involved.html#extending-the-bids-specification)
```
Comment (ericearl) [inline docs/guide.md:17]:
```suggestion
a list of proposed/ongoing BEPs can be found [here]https://bids.neuroimaging.io/get_involved.html#extending-the-bids-specification)
```

File: docs/guide.md [hunk 4]
```diff
@@ -1,9 +1,31 @@
...
+
+
+
+**_Note:_** 
+> We strongly suggest that you explore the [specification](bids-specification.readthedocs.io) and 
+> the [lists](https://bids.neuroimaging.io/get_involved.html#extending-the-bids-specification) linked [above](#what-is-a-bids-extension-proposal-(bep)) to find existing or ongoing efforts that may support what you
+are trying to add into the BIDS Specification. Someone may have already done all the work for you.
```
Comment (ericearl) [inline docs/guide.md:24]: I feel like the guide should list things out step by step, and this note should be a loud step one. That would address our open issue as well.

File: docs/guide.md (hunk 1 above)
Comment (bendhouseart) [reply docs/guide.md:3]: Need those for links, see ` [above](#what-is-a-bids-extension-proposal-(bep))`

File: docs/guide.md (hunk 4 above)
Comment (bendhouseart) [reply docs/guide.md:24]: Okay, I already put a similar warning in the about section, I'll move this warning in `docs/guide.md` to step 1 down below.

**Pull request merged**